x/application/types/errors.go

```diff
@@ -3,10 +3,13 @@ package types
 // DONTCOVER
 
 import (
-	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	errorsmod "cosmossdk.io/errors"
```

Comment:
Is there a particular reason for using the `errorsmod` alias over `sdkerrors`? Personally, I think  `sdkerrors` reads a little more cleanly.

Reply:
Ugh; personally, not a fan but I don't feel strongly about it.